numberformat/diff.emu

```diff
@@ -1592,7 +1592,12 @@
         1. Let _text_ be ! StringToCodePoints(_str_).
         1. Let _literal_ be ParseText(_text_, |StringNumericLiteral|).
         1. If _literal_ is a List of errors, return ~not-a-number~.
-        1. Return the StringIntlMV of _literal_.
+        1. Let _intlMV_ be the StringIntlMV of _literal_.
+        1. If _intlMV_ is a mathematical value and _intlMV_ &gt; ℝ(Number.MAX_VALUE), the implementation may set _intlMV_ to ~positive-infinity~.
```

Comment:
Nit: % to refer to the original value of `Number.MAX_VALUE`. (Also for the other references below.)

```suggestion
        1. If _intlMV_ is a mathematical value and _intlMV_ &gt; ℝ(%Number.MAX_VALUE%), the implementation may set _intlMV_ to ~positive-infinity~.
```